### PR TITLE
fix(rustserver): fix wipe for player.states.*.db

### LINKED ISSUE #3064: [BUG] Rust game server wipe commands not removing player.states.*.db files
## User Story

The ~/rustserver wipe and ~/rustserver full-wipe are skipping over the player.states.*.db files located in ~/serverfiles/server/rustserver/ (same place as other map specific files)

## Basic info

* **Distro:** [centos7.8]
* **Game:** [Rust]
* **Command:** [wipe]
* **Command:** [full-wipe]
* **LinuxGSM version:** [v20.5.0]

## Further Information

Just need to add the player.states.*.db files to be deleted when using the wiping / full wiping commands.

## To Reproduce

Steps to reproduce the behaviour:
1. ls -la ~/serverfiles/server/rustserver/ | grep player.states
2. wipe server or full-wipe server using wipe command
3. ls -la ~/serverfiles/server/rustserver/ | grep player.states
4. note that player.states.*.db files were NOT deleted

## Expected behaviour
1. ls -la ~/serverfiles/server/rustserver/ | grep player.states
2. wipe server or full-wipe server using wipe command
3. ls -la ~/serverfiles/server/rustserver/ | grep player.states
4. note that player.states.*.db files are gone

### PR DESCRIPTION
**Pull request by h3o66**
# Description

Includes the player.states.*.db files in the wipe

Fixes #3064

## Type of change

* [x] Bug fix (a change which fixes an issue).
* [ ] New feature (change which adds functionality).
* [ ] New Server (new server added).
* [ ] Refactor (restructures existing code).
* [ ] Comment update (typo, spelling, explanation, examples, etc).

## Checklist

PR will not be merged until all steps are complete.

* [x] This pull request links to an issue.
* [x] This pull request uses the `develop` branch as its base.
* [x] This pull request Subject follows the Conventional Commits standard.
* [x] This code follows the style guidelines of this project.
* [x] I have performed a self-review of my code.
* [x] I have checked that this code is commented where required.
* [x] I have provided a detailed with enough description of this PR.
* [ ] I have checked If documentation needs updating.

## Documentation

If documentation does need updating either update it by creating a PR (preferred) or request a documentation update.
* User docs: https://github.com/GameServerManagers/LinuxGSM-Docs
* Dev docs: https://github.com/GameServerManagers/LinuxGSM-Dev-Docs

**Thank you for your Pull Request!**

### CODE REVIEW

**Comment by dgibbs64** [comment]
This will be a hotfix

File: lgsm/functions/command_wipe.sh [hunk 1]
```diff
@@ -146,6 +146,19 @@ fn_wipe_server_files(){
 		fn_sleep_time
 		fn_script_log_pass "No player death to remove"
 	fi
+	# Wipe player sates files
```
Comment (dgibbs64) [inline lgsm/functions/command_wipe.sh:149]: typo

File: lgsm/functions/fix_steamcmd.sh [hunk 2]
```diff
@@ -79,6 +79,8 @@ elif [ "${shortname}" == "cmw" ]; then
 	fn_fix_steamclient_so "32" "${executabledir}/lib"
 elif [ "${shortname}" == "cs" ]; then
 	fn_fix_steamclient_so "32" "${serverfiles}"
+elif [ "${shortname}" == "ins" ]; then
```
Comment (dgibbs64) [inline lgsm/functions/fix_steamcmd.sh:82]: this is already merged in to develop. Please remove as not part of the hotfix

**Review by dgibbs64 — CHANGES_REQUESTED** [review]
Two minor changes then ready to merge

File: lgsm/functions/fix_steamcmd.sh (hunk 2 above)
Comment (h3o66) [reply lgsm/functions/fix_steamcmd.sh]: rebases on master only with a single commit

File: lgsm/functions/command_wipe.sh (hunk 1 above)
Comment (h3o66) [reply lgsm/functions/command_wipe.sh]: fixed typo

**Pull request merged**